Makefile

```diff
@@ -5,6 +5,7 @@ BINDIR = $(PREFIX)/bin
 # test defaults
 TEST = 00basic
 STAGE = gentoo/stage3-amd64
+STAGE_TAG = latest
```

Comment:
Could you put a known-good tag here? If in doubt, just take the date from last commit passing CI.

Reply:
Oh, I hadn't thought of that, great idea ;)

Updated in the latest version.